frontend/src/components/PasswordRecovery/PasswordRecovery.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+
+
+interface PasswordRecovery {
+    abortFunction: (
+        event: React.MouseEvent<HTMLButtonElement, MouseEvent>
+    ) => void;
+}
+
+const PasswordRecovery: React.FunctionComponent<PasswordRecovery> = ({
+                                                                         abortFunction
+                                                                     }: PasswordRecovery) => {
+    const [emailAddress, setEmailAddress] = React.useState("");
+
+    const alertMessage = `E-Mail Address: ${emailAddress}`;
+
+    function handleSubmit() {
+        console.log("Clicked");
+
+        alert(alertMessage);
+    }
+
+    return (
+        <div>
+            <Form onSubmit={handleSubmit}>
+                <Form.Group controlId="formRecoveryEmail">
+                    <Form.Label>E-Mail Adresse</Form.Label>
+                    <Form.Control as="input" value={emailAddress} onChange={(e) => setEmailAddress(e.currentTarget.value)} placeholder="E-Mail Adresse"/>
```

Comment:
We don't have any validation on this function as of yet, do we want to add that?

Reply:
absolutely! some kind of form validation needs to be implemented. feel free to add a feature request as an issue, otherwise i can do it later in the day.